Orange/data/variable.py

```diff
@@ -622,7 +625,7 @@ def __init__(
             *, sparse=False
     ):
         """ Construct a discrete variable descriptor with the given values. """
-        values = tuple(values)  # some people (including me) pass a generator
+        values = TupleList(values)  # some people pass a generator
```

Comment:
Correct me if I'm wrong, but I think this can remain just `tuple` (for creating new instances) as long as the above is there for unpickling of saved older classes.

Reply:
You are right :)